### ISSUE TITLE
Give outdir a better default value in build/circle-test.sh

### ISSUE THREAD

**Issue by yaojingguo** [cockroachdb/cockroach#2984]
If both `CIRCLE_ARTIFACTS` and `TMPDIR` are unset, `outdir` is set to a empty string. The following `${outdir}/slow.txt` will be evaluated to `/slow.txt`. It is common that a non-root user does not have the permissions to create files in `/`. I am using Ubuntu 14.04. When I run `./build/circle-local.sh`, there will be the following error:

```
gofmt (simplify)
goimports
./build/circle-test.sh: line 57: /slow.txt: Permission denied
find: cannot search `': No such file or directory
```

A better default value instead of a empty string should be given.

**Comment by tschottdorf** [comment]
Sounds reasonable - would you be willing to implement that? Defaulting to `/tmp` should be a sane default for now.

**Comment by yaojingguo** [comment]
Happy to work on it.

**Comment by tamird** [comment]
Closed by #2990.
